runtime/gc_base/GCExtensions.hpp

```diff
@@ -356,7 +356,7 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 		, minimumFreeSizeForSurvivor(DEFAULT_SURVIVOR_MINIMUM_FREESIZE)
 		, freeSizeThresholdForSurvivor(DEFAULT_SURVIVOR_THRESHOLD)
 		, recycleRemainders(true)
-		, continuationListOption(verify_continuation_list)
+		, continuationListOption(enable_continuation_list)
```

Comment:
From an in-person discussion, I understand that the last unmount (that would normally release native resource)  is not really guarantied as previously believed (is this really a consequence of this redesign, or it was true all along?), so GC pruning pass is still needed. 

This is unfortunate, since this list contains all Continuation objects allocated since previous GC, hence mostly dead by next GC and since most of VTs realistically end up doing last unmount. Doing this pass every GC for those exception cases is not good... Pauses for low pause GCs (like Concurrent Scavenger) may noticeably increase.

We will probably need to optimize the list further. For example, maintain only Continuations that have not finished, by removing them from the list during last unmount.  Would require the list to be double linked. Contention should not be much of any issue since we use multiple lists. @LinHu2016  

Some more advanced things may be considered as well with better understanding under which circumstances VT/continuation can die without doing last unmount (which understanding I don't have). But, for example, if it's because Scheduler died and all its active VTs, we should just process those.

Reply:
To answer my own question (after another discussion with @fengxue-IS and @babsingh), seems like the scenario of VTs dying before doing last unmount was possible even in current workloads that we use, but since the VT list served as strong root, and for a potential VT being added on first mount, dying but never removed, it would  be kept alive forever. Not only GC would not be able to see it as a dead object (with dangling stack) and trigger a verification assert, but this scenario would also cause leak in both java heap and native memory.

Now that the VT list is removed, such a VT can really die and the assert triggers, hence enabling  GC list processing must be done as a part of this change.

Reply:
Is the continuation list actually optional now that we're using it instead of the vthread list?

Reply:
With this change, continuation list must be enabled to correctly perform JIT codecache clean up and JVMTI operations